servicer test failure

```
======================================================================
FAIL: testUserDefinedModel (master.servicer_test.ServicerTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/l.zou/git/elasticdl/elasticdl/master/servicer_test.py", line 226, in testUserDefinedModel
    'dense_1/kernel:0'], list(sorted(model.param.keys())))
AssertionError: Lists differ: ['dense/bias:0', 'dense/kernel:0', 'dense_1/bias:0', 'dense_1/kernel:0'] != ['dense_1/bias:0', 'dense_1/kernel:0', 'dense_2/bias:0', 'dense_2/kernel:0']

First differing element 0:
'dense/bias:0'
'dense_1/bias:0'

- ['dense/bias:0', 'dense/kernel:0', 'dense_1/bias:0', 'dense_1/kernel:0']
+ ['dense_1/bias:0', 'dense_1/kernel:0', 'dense_2/bias:0', 'dense_2/kernel:0']

----------------------------------------------------------------------
Ran 11 tests in 18.707s
```

--- a/elasticdl/master/servicer_test.py
+++ b/elasticdl/master/servicer_test.py
@@ -222,5 +222,5 @@ class ServicerTest(unittest.TestCase):
             master.set_model_var(variable.name, variable.numpy())
         model = master.GetModel(req, None)
         self.assertEqual(0, model.version)
-        self.assertEqual(['dense/bias:0', 'dense/kernel:0', 'dense_1/bias:0', 
-            'dense_1/kernel:0'], list(sorted(model.param.keys())))
+        self.assertEqual(['dense_1/bias:0', 'dense_1/kernel:0', 'dense_2/bias:0', 
+            'dense_2/kernel:0'], list(sorted(model.param.keys())))

